proto-lens-protoc/src/Data/ProtoLens/Compiler/Generate.hs

```diff
@@ -683,23 +692,36 @@ plainRecordField env (PlainFieldInfo kind f) = case kind of
         -- data Foo = Foo { _Foo_bar :: [Bar] }
         -- type instance Field "bar" Foo = [Bar]
     RepeatedField {} ->
-            recordField listType
-                  [LensInstance
+            recordField vectorType
+                  [ LensInstance
                       { lensSymbol = baseName
                       , lensFieldType = listType
+                      , lensExp = vectorAccessor
+                      }
+                  , LensInstance
+                      { lensSymbol = "vector'" <> baseName
```

Comment:
Yeah, bikeshedding seems reasonable.  I'm against a prefix as short as `v`, but `vec` seems pretty reasonable.

I think we should also consider a migration path for eventually deprecating the list lenses and making the `baseName` lens provide a vector.  We can probably punt on it for now though.

Another option I was thinking about was `baseName <> "'vector"`, though that's less consistent with the `maybe'*` lenses.

I think we should *not* CamelCase `baseName`.  First, because it would be different behavior than the `maybe'*` lenses.  And second, because the `'` already acts as a separator so the upper-casing would be redundant and make it look like a constructor rather than a value.

Reply:
The list deprecation could be achieved with some kind of option people stick into their .proto. But's a battle for another day.

Reply:
Looks like we agreed to `vec`.

Reply:
Done.